### PR TITLE
Fetch resume reviews on student app

### LINKED ISSUE #186: Integrate fetch resume review API with backend for student

### PR DESCRIPTION
**Pull request by armiantos**
# Overview

* Fetch resume review objects when the user lands on the resume review page as a student

# Changes

* Implemented getMyResumeReviews thunk
* Integrated thunk with resumeReviewSlice

# Questions & Notes

* This will unblock #181 

# Issue tracking

Closes #186 

# Testing & Demo

Refer to unit tests

### CODE REVIEW

File: www/src/redux/substores/student/slices/resumeReviewSlice.ts [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
-        currentResume: null,
-    },
+    initialState,
     reducers: {},
+    extraReducers: (builder) => {
+        builder.addCase(getMyResumeReviews.pending, (state) => {
+            state.isLoading = true;
+        });
+        builder.addCase(getMyResumeReviews.fulfilled, (state, action) => {
+            state.resumeReviews = action.payload.resumeReviews;
```
Comment (calebschoepp) [inline www/src/redux/substores/student/slices/resumeReviewSlice.ts:25]: Should this set loading to false?

File: www/src/routes/student/ResumeReview.tsx [hunk 2]
```diff
@@ -1,10 +1,26 @@
...
+    const { resumeReviews } = useStudentSelector((state) => state.resumeReview);
+    const { getAccessTokenSilently } = useAuth0();
+    const dispatch = useStudentDispatch();
+
+    useEffect(() => {
+        dispatch(getMyResumeReviews({ tokenAcquirer: getAccessTokenSilently }));
+    }, []);
+
+    useEffect(() => {
+        // TODO: display upload button if there are no resume reviews
```
Comment (calebschoepp) [inline www/src/routes/student/ResumeReview.tsx:20]: Wouldn't this TODO be in the html you return?

**Review by calebschoepp — APPROVED** [review]
Couple questions but pretty much LGTM

File: www/src/redux/substores/student/slices/resumeReviewSlice.ts (hunk 1 above)
Comment (armiantos) [reply www/src/redux/substores/student/slices/resumeReviewSlice.ts:25]: Good catch, will update and merge 😀

File: www/src/routes/student/ResumeReview.tsx (hunk 2 above)
Comment (armiantos) [reply www/src/routes/student/ResumeReview.tsx:20]: Yes this is cleaned up in #185

**Pull request merged**